src/app.cfg

```diff
@@ -1,2 +1,2 @@
-/home/nlesquoy/ghq/github.com/LaTeX-Project-Manager/settings/config.json
+/home/nlesquoy/ghq/github.com/iScsc/LaTeX-Project-Manager/settings/config.json
```

Comment:
i think such a file should go to `./res/` or `./config/` in the root of the repo, not in `./src/` :wink: